.github/workflows/deploy.yml

```diff
@@ -25,18 +21,14 @@ on:
         required: false
         default: "postgres_db"
         type: string
+      apply_network_policy:
+        required: false
+        default: false
+        type: boolean
       deploy_to_dev:
         required: false
         default: true
         type: boolean
-      spawn-rate: 
```

Comment:
Line 32-39 are fine to remove @gio-karunakaran 😄 as we have it in the shared-test.yml file now.